yaml/boundaries.yaml

```diff
@@ -175,20 +171,14 @@ filters:
       <<: *output_properties
       kind: county
     table: osm
-  # this filter covers admin_level 3 boundaries for countries and states,
-  # turning them into disputed_reference_lines
-  - filter:
-      all:
-        - geom_type: line
-        - admin_level: '3'
-        - any:
-            - place: 'state'
-            - place: 'country'
+  # These are kind=disputed_reference_lines that, for viewpoint XX in disputed_by,
+  # becomes kind:xx=unrecognized_disputed_reference_line
```

Comment:
Expand comment that the LUA preprocessing will project admin_level 3 and 5 to appropriate tags for us, so we don't need to consider them directly there.

Reply:
fixed in 9506b97782500824588c5745c04dc520b202da17